statsmodels/stats/proportion.py

```diff
@@ -245,7 +245,7 @@ def _power_ztost(mean_low, var_low, mean_upp, var_upp, mean_alt, var_alt,
     #print mean_upp, np.sqrt(var_upp), crit, var_upp
     if np.any(k_low > k_upp):   #vectorize
         import warnings
-        warnings.warn("no overlap, power is zero", UserWarning)
+        warnings.warn("no overlap, power is zero", NoPowerWarning)
```

Comment:
`NoPowerWarning` sounds too specific. We need a warning that is a bit more generic where the more specific information is only in the message